main.spec

```diff
@@ -0,0 +1,40 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+
+block_cipher = None
+
+
+a = Analysis(['main.py'],
+             pathex=[],
+             binaries=[],
+             datas=[('./assets/fonts/*.ttf', 'assets/fonts'), ('./assets/graphics/*.png', 'assets/graphics'), ('./assets/graphics/explosion/*.png', 'assets/graphics/explosion'), ('assets/sounds/*.wav', 'assets/sounds')],
+             hiddenimports=[],
+             hookspath=[],
+             hooksconfig={},
+             runtime_hooks=[],
+             excludes=[],
+             win_no_prefer_redirects=False,
+             win_private_assemblies=False,
+             cipher=block_cipher,
+             noarchive=False)
+pyz = PYZ(a.pure, a.zipped_data,
+             cipher=block_cipher)
+
+exe = EXE(pyz,
+          a.scripts,
+          a.binaries,
+          a.zipfiles,
+          a.datas,  
+          [],
+          name='main',
+          debug=False,
+          bootloader_ignore_signals=False,
+          strip=False,
+          upx=True,
+          upx_exclude=[],
+          runtime_tmpdir=None,
+          console=False,
+          disable_windowed_traceback=False,
+          target_arch=None,
+          codesign_identity=None,
+          entitlements_file=None )
```

Comment:
@rusty-electron If more assets get included in the  codebase then what things needs to get changed in the spec file?

Reply:
We need to add the path to the new assets in the `datas` variable as a tuple of the form (probably with glob matching if many files exist in a single directory): `('path with respect to parent dir', 'same path with the filename stripped')`. I will provide more details on discord soon.